pkg/config/containers.conf

```diff
@@ -336,6 +336,11 @@ default_sysctls = [
 #
 # lock_type** = "shm"
 
+# Indicates if Podman is running inside a VM via Podman Machine.
+# Podman uses this value to do extra setup around networking from the
+# container inside the VM to to host.
+# machine_enabled=false
```

Comment:
Could you please add two spaces? 
like this: `machine_enabled = false`

Reply:
Let's deal with that in a separate PR.

Reply:
@rhatdan When will the process begin?

Reply:
You can open a PR now, if you want.

Reply:
ohhh